### PR TITLE
Fix leaderboard item have token undefined

### PR DESCRIPTION
**Pull request by nguyenhuudungz**

### CODE REVIEW

**Review by nguyenhoaidanh — APPROVED** [review]

File: src/pages/Campaign/LeaderboardLayout.tsx [hunk 1]
```diff
@@ -113,7 +113,7 @@ export default function LeaderboardLayout({
         </LeaderboardTableBodyItem>
         {showRewardsColumn && (
           <LeaderboardTableBodyItem align="right" isThisRankingEligible={isThisRankingEligible}>
-            {data.rewardAmount.toSignificant(DEFAULT_SIGNIFICANT)} {data.token.symbol}
+            {data.rewardAmount.toSignificant(DEFAULT_SIGNIFICANT)} {data?.token?.symbol}
```
Comment (namgold) [inline src/pages/Campaign/LeaderboardLayout.tsx:116]: should be `{data?.token?.symbol || ''}`

Comment (nguyenhuudungz) [reply src/pages/Campaign/LeaderboardLayout.tsx:116]: It doesn't render `undefined` to the screen but it is better for the screen reader.

Comment (namgold) [reply src/pages/Campaign/LeaderboardLayout.tsx]: Im not sure in screen reader case. How does it work in this case? @nguyenhuudungz

Comment (nguyenhuudungz) [reply src/pages/Campaign/LeaderboardLayout.tsx]: image

**Pull request merged**